apps/web/src/pages/brand/tabs/BrandingForm.tsx

```diff
@@ -82,10 +88,12 @@ export function BrandingForm() {
     setValue('image', path);
   }
 
+  const dropzoneRef = useRef<() => void>(null);
+
   async function saveBrandsForm({ color, fontFamily, image }) {
     const brandData = {
       color,
-      logo: image,
+      logo: image || null,
```

Comment:
"removed" `image` is empty string, but backend has validation rule for strings to check if it's URL.

as `logo` is optional, `null` will pass that validation rule and successfully remove the uploaded logo.